pkg/innerring/processors/netmap/processor.go

```diff
@@ -66,7 +66,8 @@ type (
 		handleAuditSettlements event.Handler
 		handleAlphabetSync     event.Handler
 
-		nodeValidator NodeValidator
+		nodeLocodeValidator  NodeValidator
```

Comment:
I suggest to not split up validators as long as they are required to process the same event. Instead, we can merge the validators into one complex.

Reply:
added `CompositeValidator`